tm2/pkg/p2p/transport_test.go

```diff
@@ -122,14 +122,12 @@ func TestMultiplexTransport_Accept(t *testing.T) {
 
 		transport := NewMultiplexTransport(ni, nk, mCfg, logger)
 
-		p, err := transport.Accept(context.Background(), nil)
+		ctx, cancel := context.WithTimeout(context.Background(), 100*time.Millisecond)
+		defer cancel()
 
+		p, err := transport.Accept(ctx, nil)
 		assert.Nil(t, p)
-		assert.ErrorIs(
-			t,
-			err,
-			errTransportInactive,
-		)
+		assert.ErrorIs(t, err, context.DeadlineExceeded)
```

Comment:
looks good but gives me a feeling that we are missing at least a new test.

Reply:
Updated, the comment should better match the behaviors.

Reply:
Added some new tests.